src/Chip/common/ChipRemovable.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { CloseIcon } from "../../icons";
+import { Chip, ChipProps } from "../Chip";
+import { ChipAdornment } from "../ChipAdornment";
+
+export interface ChipRemovableProps extends ChipProps {
+  onRemove?: () => void;
```

Comment:
```suggestion
  onRemove: () => void;
```
let's make it required, if a developer wants to omit it then he should either use the standard Chip or this one should have some kind of disabled prop.

Reply:
yup, that's a good idea 👍🏻